maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -103,94 +93,11 @@ private void testing(File runFile) throws MojoExecutionException, Exception, IOE
 			}
 
 			if (resolve) {
-				resolve(run);
+				run.resolve(failOnChanges);
```

Comment:
So the change would be whether the resolve configuration flag results in mutating the bndrun source file. That seems fairly minor and can be documented as a breaking change. We can still honor the resolve flag, but it won't result in changing the bndrun source file.